Google Fonts/decompose-transformed-components.py

```diff
@@ -0,0 +1,36 @@
+#MenuTitle: Decompose Transformed Components
+"""TTFautohint doesn't like components which have been flipped"""
+
+def find_transformed_component_glyphs(font):
+	found = set()
+	for idx, master in enumerate(font.masters):
+		for glyph in font.glyphs:
+			components = glyph.layers[idx].components
+			if len(components) >= 2:
+				continue
```

Comment:
@m4rc1e I wonder, why skipping composite glyphs with 2 or more components?

Reply:
My bad. They should all be decomposed imo.

I guess this is there because one of the sources I encountered had a parenright which was flipped component of a parentleft.